components/servo/fake-ld.sh

```diff
@@ -9,7 +9,25 @@ set -o nounset
 set -o pipefail
 
 TARGET_DIR="${OUT_DIR}/../../.."
-arm-linux-androideabi-gcc "${@}" \
-                          "${LDFLAGS-}" -lc -shared \
-                          -o "${TARGET_DIR}/libservo.so"
-touch "${TARGET_DIR}/servo"
+
+export _ANDROID_ARCH=arch-arm
+export ANDROID_SYSROOT="$ANDROID_NDK/platforms/android-18/$_ANDROID_ARCH"
+export _ANDROID_EABI=arm-linux-androideabi-4.9
+ANDROID_TOOLCHAIN=""
+for host in "linux-x86_64" "linux-x86" "darwin-x86_64" "darwin-x86"
```

Comment:
style nit: use a `; do` at the end of this line instead of putting it on the next line